docs/spec/openapi.yml

```diff
@@ -10932,3 +11168,88 @@ components:
           description: A machine-readable description of the cause of the error. If this
             value is empty there is no information available.
           type: string
+    # ---------- Engine Features Components -------------------------
+    SplitHorizonDNSConfig:
+      description: SplitHorizonDNSConfig is the Schema for the Split-Horizon DNS engine feature for PSMDB.
+      properties:
+        apiVersion:
+          description: |-
+            APIVersion defines the versioned schema of this representation of an object.
+            Servers should convert recognized schemas to the latest internal value, and
+            may reject unrecognized values.
+            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+          type: string
+        kind:
+          description: |-
+            Kind is a string value representing the REST resource this object represents.
+            Servers may infer this from the endpoint the client submits requests to.
+            Cannot be updated.
+            In CamelCase.
+            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: Spec defines the desired state of SplitHorizonDNSConfig
+          properties:
+            baseDomainNameSuffix:
+              description: |-
+                BaseDomainNameSuffix is the base domain name suffix for generating domain names for each Pod in ReplicaSet.
+                It should be a valid domain name suffix.
+              example: example.com
+              minLength: 3
+              pattern: ^([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$
+              type: string
+            tls:
+              description: TLS is the TLS configuration for the split-horizon DNS
+                configuration.
+              properties:
+                certificate:
+                  description: Certificate is the TLS certificate and key for the
+                    split-horizon DNS configuration.
+                  properties:
+                    caCertFile:
+                      description: CaCertFile is based64 encoded CA certificate
+                        file content.
+                      type: string
+                    certFile:
```

Comment:
The spec is not identical to what is generated now in the `everest-operator` repo, please keep it in sync before merging to avoid having unrelated updates in the spec in other PRs